### ISSUE TITLE
Add a quick-copy button next to the "Share Message" context menu option

### ISSUE THREAD

**Issue by turt2live** [element-hq/element-web#7187]
Add a quick-copy button next to the "Share Message" context menu option so I don't have to click into the dialog, and because I keep forgetting to right click -> copy link

<img width="178" alt="" src="https://user-images.githubusercontent.com/558581/133342282-2c6e96fe-f8d9-482f-9de1-bc532a2e68b6.png">

**Comment by lampholder** [comment]
You mean like a little ![image](https://user-images.githubusercontent.com/1922197/44353692-ac2a2a80-a49f-11e8-94ac-f95eef3fc554.png) next to the 'Share message' text in the context menu?

**Comment by turt2live** [comment]
Yup, although how that would look/be designed is probably a harder problem.

**Comment by ArneNostitz** [comment]
for the Desktop app it would be great to have a shortcut for this as well. (also directly linking to the element app with the link if that is possible would be a great option). so selecting msg; pressing shortcut = copy direct link to msg into clipboard ... no menu & mouse clicking needed

**Comment by TemoCrevlis** [comment]
I've only used that share button to copy the message link.
Would just change the share button into a copy-message-link button or copy-event-link.
